workflow/rules/qc.smk

```diff
@@ -133,7 +148,8 @@ rule extract_nonhuman_reads:
         (samtools view  -@ {threads} -u -f 4 -F 264 {input} > {output.t1}) 2> {log}
         (samtools view  -@ {threads} -u -f 8 -F 260 {input} > {output.t2}) 2>> {log}
         (samtools view  -@ {threads} -u -f 12 -F 256 {input} > {output.t3}) 2>> {log}
-        samtools merge -@ {threads} -n {output.unsorted} {output.t1} {output.t2} {output.t3} >> {log} 2>&1
+        (samtools view  -@ {threads} -h -f 1 -F 12 {input} | grep 'NC_045512.2' | samtools view -u > {output.t4}) 2> {log}
```

Comment:
It seems when filtering for the covid reads with Samtools view, the 

> `-f 2` (PROPER_PAIR: each segment properly aligned according to the aligner)

 is too strict on the test reads.

When using the flag 

> `-f 1` (PAIRED: paired-end (or multiple-segment) sequencing technology) 

more covid reads of the test data are accpeted.